source/agora/consensus/protocol/Nominator.d

```diff
@@ -156,6 +156,9 @@ public extern (C++) class Nominator : SCPDriver
     /// User configured nomination frequency
     private const Duration nomination_interval;
 
+    /// User configured nomination frequency
+    private Duration round_timeout;
```

Comment:
If we use this new variable for nomination timing, we should remove the `nomination_interval` variable. There's no use for the old variable as far as I can see.

Reply:
We need it to reset the `round_timeout` to the configured value from `config.validator.nomination_interval` which is passed into the `Nominator` class when constructed.